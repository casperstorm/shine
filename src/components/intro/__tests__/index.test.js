import renderer from 'react-test-renderer'
import React from 'react'

import Intro from '../index'

describe('intro', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Intro onEnd={() => {}} />)
    expect(tree).toMatchSnapshot()
  })
})
