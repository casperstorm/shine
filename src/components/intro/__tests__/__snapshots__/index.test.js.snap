// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`intro should render correctly 1`] = `
<RCTImageSequence
  framesPerSecond={24}
  images={
    Array [
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
      Object {
        "__packager_asset": true,
        "height": 100,
        "scale": 2,
        "uri": "assets/full/path/to/directory/icon@2x.png",
        "width": 100,
      },
    ]
  }
  startFrameIndex={0}
  style={
    Object {
      "height": 300,
      "width": 300,
    }
  }
/>
`;
